moja-kuhinjica-frontend/src/pages/mealReservation/index.tsx

```diff
@@ -201,6 +208,15 @@ const MealReservation = (): JSX.Element => {
         setConfirmationModalIsOpen(false)
     }
 
+    
+   const isBookingAllowed = (): boolean => {
+       const today = new Date();
+       const activeDay = new Date(activeDate.split('/').reverse().join('/'));
+        const currentHour = today.getHours();
+        const cutoffHour = 10;
+        return !(currentHour >= cutoffHour && today.toDateString() === activeDay.toDateString())
```

Comment:
currentHour >== cutOffHour 
and can you try different method to compare today and activeDay
try in dayJS